src/cmd/services/m3dbnode/config/config.go

```diff
@@ -76,25 +100,25 @@ type DBConfiguration struct {
 	Transforms TransformConfiguration `yaml:"transforms"`
 
 	// Logging configuration.
-	Logging xlog.Configuration `yaml:"logging"`
+	Logging *xlog.Configuration `yaml:"logging"`
```

Comment:
Thoughts on avoiding pointers in config where it makes sense? e.g., `len(str)` and `*string == nil` are effectively equivalent checks, same with struct equivalence (against a zero value), with the benefit of not introducing another zero-like state that needs to be validated as well.

What do you think?

Reply:
Unfortunately it's much harder to check if zero struct or not.